RecoBTag/Combined/python/combinedMVA_cff.py

```diff
@@ -11,6 +11,16 @@
 from RecoBTag.Combined.pfNegativeCombinedMVABJetTags_cfi import *
 from RecoBTag.Combined.pfPositiveCombinedMVABJetTags_cfi import *
 
-# CombinedMVA V2
+#combined MVA V2
+from RecoBTag.Combined.combinedMVAV2Computer_cfi import *
```

Comment:
Here you should only add JetTag producers. All the computers need to go into `RecoBTag/Combined/python/combinedMVA_EventSetup_cff.py` (see #12848 for reference).

Also, don't forget to add this PR to https://twiki.cern.ch/twiki/bin/view/CMS/BTagSoftwarePullRequests Thanks